Resetting database no longer works

In `WooWellSqlUtils` we [have a function to reset the database](https://github.com/woocommerce/woocommerce-android/blob/a81e857d177574e66ba1bfb3ae19d53756e307c6/WooCommerce/src/main/kotlin/com/woocommerce/android/WooWellSqlConfig.kt#L51). This is useful when you want to clear the DB without clearing storage and your login info.

Unfortunately, this no longer works.

```
Caused by: kotlin.UninitializedPropertyAccessException: lateinit property context has not been initialized
        at com.woocommerce.android.AppPrefs.getPreferences(AppPrefs.kt:384)
        at com.woocommerce.android.AppPrefs.setBoolean(AppPrefs.kt:382)
        at com.woocommerce.android.AppPrefs.setDatabaseDowngraded(AppPrefs.kt:265)
        at com.woocommerce.android.WooWellSqlConfig.resetDatabase(WooWellSqlConfig.kt:60)
```

--- a/WooCommerce/src/main/kotlin/com/woocommerce/android/WooCommerce.kt
+++ b/WooCommerce/src/main/kotlin/com/woocommerce/android/WooCommerce.kt
@@ -142,6 +142,9 @@ open class WooCommerce : MultiDexApplication(), HasAndroidInjector, ApplicationL
             null
         }
 
+        // Developers can uncomment the line below to clear the db tables at startup
+        // wellSqlConfig.resetDatabase()
+
         notificationHandler.createNotificationChannels(this)
 
         val lifecycleMonitor = ApplicationLifecycleMonitor(this)

--- a/WooCommerce/src/main/kotlin/com/woocommerce/android/WooWellSqlConfig.kt
+++ b/WooCommerce/src/main/kotlin/com/woocommerce/android/WooWellSqlConfig.kt
@@ -45,8 +45,8 @@ class WooWellSqlConfig(context: Context) : WellSqlConfig(context, ADDON_WOOCOMME
 
     /**
      * Useful during development when we want to test features with a "fresh" database. This can be
-     * called from WooCommerce.onCreate() after we initialize the database. For safety, this has no
-     * effect when called from a release build.
+     * called from WooCommerce.onCreate() after we initialize the database and access the selected
+     * site. For safety, this has no effect when called from a release build.
      */
     fun resetDatabase() {
         if (BuildConfig.DEBUG) {

--- a/WooCommerce/src/main/kotlin/com/woocommerce/android/WooWellSqlConfig.kt
+++ b/WooCommerce/src/main/kotlin/com/woocommerce/android/WooWellSqlConfig.kt
@@ -57,7 +57,10 @@ class WooWellSqlConfig(context: Context) : WellSqlConfig(context, ADDON_WOOCOMME
             )
             toast.setGravity(Gravity.CENTER_HORIZONTAL or Gravity.BOTTOM, 0, 0)
             toast.show()
+
+            AppPrefs.init(context)
             AppPrefs.setDatabaseDowngraded(true)
+
             reset()
         }
     }

